docs/connected_orgs.rst

```diff
@@ -1,14 +1,95 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you first need to have a Connected App configured in a persistent Salesforce org.
 
-Connect a production org
-------------------------
 
-Connect a sandbox org
+
+CumulusCI's Connected App
+-------------------------
+CumulusCI comes with a connected app that you can deploy to any org in your ``sfdx`` keychain.
+By default, this this deploy to the org configured as the ``defaultdevhubusername``::
+Use the following to deploy the CumulusCI's connected app to an org::
+
+    $ cci task run connected_app
+
+This command also takes care to setup CumulusCI's ``connected_app`` service in the keychain for you.
+If you want to see the information for the connected app, you can view it with::
+
+    $ cci service info connected_app
+
+
+
+
+The ``org connect`` Command
+---------------------------
+To connect to a persistent org use::
+
+    $ cci org connect <org_name>
+
+This will automatically open a browser window pointed to a Salesforce login page.
+The provided ``<org_name>`` will be the name that CumulusCI associates with org you log into.
+
+
+
+Production and Developer Edition Orgs
```

Comment:
Could we collapse these subsections and just describe when you need the `--sandbox` flag inline?

`--sandbox` is also needed if you're connecting to a scratch org you didn't create.